libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/encryption/EncryptionService.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.libraries.matrix.api.encryption
+
+import kotlinx.coroutines.flow.StateFlow
+
+interface EncryptionService {
+    val backupStateStateFlow: StateFlow<BackupState>
+    val recoveryStateStateFlow: StateFlow<RecoveryState>
+    val backupUploadStateStateFlow: StateFlow<BackupUploadState>
+    val enableRecoveryProgressStateFlow: StateFlow<EnableRecoveryProgress>
+
+    suspend fun enableBackups(): Result<Unit>
```

Comment:
Is there any way to disable them?

Reply:
Yes, there is one, but the name from the SDK is `disableRecovery`, do not ask me why. I will document the API later.
We have to decide if we stick to the SDK name or if we change it to make is clearer.